.NET/Microsoft.Recognizers.Definitions.Common/Portuguese/NumbersWithUnitDefinitions.cs

```diff
@@ -722,7 +723,7 @@ public static class NumbersWithUnitDefinitions
             { @"Decímetro", @"dm|decimetro|decímetro|decimetros|decímetros" },
             { @"Centímetro", @"cm|centimetro|centímetro|centimetros|centimetros" },
             { @"Milímetro", @"mm|milimetro|milímetro|milimetros|milímetros" },
-            { @"Micrômetro", @"µm|um|micrometro|micrômetro|micrómetro|micrometros|micrômetros|micrómetros|micron|mícron|microns|mícrons|micra" },

```

Comment:
This should not be removed as its sometimes used informally as the unit ("10 um", "mil um"). Perhaps adding it to the ambiguity list to be removed if not prefixed by a number.

Reply:
ok, I will add a filter to allow only cases where is preceded by numbers in digits. 
Being a number, it can still be common to have it prefixed by numbers without being a unit. I added such a test case already ("três dois um"), and it can appear in expressions like "vinte um", "cinqüenta um"....

Reply:
In "vinte um" and "cinqüenta um" it's part of the number. But you're right that in "três dois um" it's a problem. Hmmm.

Reply:
added support for "10 um" but not "mil um".